src/app/modules/submission/submission.service.ts

```diff
@@ -166,6 +166,112 @@ export const getProcessedResponses = (
   return ok(processedResponses)
 }
 
+export class ParsedResponsesObject {
```

Comment:
Should this be in its own file as a service? `ParsedResponseService` or something

Reply:
agreed that this object probably doesnt need to be in submission service anymore, but i was thinking that `ParsedResponsesObject` might actually be more of a dataclass than a service, in that its primary purpose is to store data and project it in different ways (similar to the `SubmissionsEmailObj` that will be replaced).

I think therefore, this object could go to `email-submission.util.ts`?